core/src/main/scala/kafka/log/Log.scala

```diff
@@ -103,20 +103,22 @@ class Log(val dir: File,
     else
       0
   }
-  val t = time.milliseconds
+
+  private[this] val t = time.milliseconds
```

Comment:
Not part of this patch, but since we're here, maybe we can choose a better name? It actually doesn't even seem like this needs to be a field since it's only used in a log message on startup. (Feel free to ignore this comment if you think it's too far out of scope)